config/settings/prod.py

```diff
@@ -9,7 +9,7 @@
     # Allow local host's IP address and hostname for health probes
     socket.gethostname(),
     socket.gethostbyname(socket.gethostname()),
-    CORGI_DOMAIN_BASE,  # noqa: F405
+    CORGI_DOMAIN,  # noqa: F405
```

Comment:
I'm a little concerned this won't work in community prod, but I guess we'll know pretty quick and be able to back it out.